### PR TITLE
🐛Fix unrelated capm3 mentions under hack folder scripts

### PR DESCRIPTION
**Pull request by furkatgofurov7**
**What this PR does / why we need it**:
Replaces unrelated capm3 mentions with the proper path in the scripts under the hack folder

### CODE REVIEW

File: hack/gofmt.sh [hunk 1]
```diff
@@ -14,9 +14,9 @@ if [ "${IS_CONTAINER}" != "false" ]; then
 else
   "${CONTAINER_RUNTIME}" run --rm \
     --env IS_CONTAINER=TRUE \
-    --volume "${PWD}:/go/src/github.com/metal3-io/cluster-api-provider-metal3:ro,z" \
+    --volume "${PWD}:/workdir:ro,z" \
     --entrypoint sh \
-    --workdir /go/src/github.com/metal3-io/cluster-api-provider-metal3 \
+    --workdir /workdir \
```
Comment (fmuyassarov) [inline hack/gofmt.sh:19]: maybe call it ipam? 
```suggestion
    --workdir /ipam \
```

Comment (furkatgofurov7) [reply hack/gofmt.sh:19]: I am not sure that would work, let me try it locally

Comment (fmuyassarov) [reply hack/gofmt.sh:19]: you need to change the mount path as well

**Review by fmuyassarov — COMMENTED** [review]
/lgtm

**Review by kashifest — APPROVED** [review]

**Pull request merged**